contracts/test/PoolModifyPositionTest.sol

```diff
@@ -22,14 +22,23 @@ contract PoolModifyPositionTest is ILockCallback {
         address sender;
         PoolKey key;
         IPoolManager.ModifyPositionParams params;
+        bytes hookData;
     }
 
     function modifyPosition(PoolKey memory key, IPoolManager.ModifyPositionParams memory params)
         external
         payable
         returns (BalanceDelta delta)
     {
-        delta = abi.decode(manager.lock(abi.encode(CallbackData(msg.sender, key, params))), (BalanceDelta));
+        return modifyPosition(key, params, new bytes(0));
```

Comment:
same as swap, I wonder if we can minimize the impact on gas by the test contracts

Reply:
I removed the old `modifyPosition()` and the gas is practically the same, or worse in some cases 🤔 

Here's the diff
https://github.com/saucepoint/v4-core/pull/1/files